client/blocks/nps-survey/recommendation-select.jsx

```diff
@@ -31,19 +32,20 @@ class RecommendationSelect extends PureComponent {
 	}
 
 	render() {
+		const { translate } = this.props;
 		const values = range( 0, 11 );
 		const options = values.map( value => this.renderOption( value ) );
 
 		return (
 			<div className="nps-survey__recommendation-select">
 				<div className="nps-survey__scale-labels">
-					<span>Unlikely</span>
-					<span className="nps-survey__very-likely-label">Very Likely</span>
+					<span>{ translate( 'Unlikely' ) }</span>
+					<span className="nps-survey__very-likely-label">{ translate( 'Very Likely' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Very likely' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).